ApplicationServerDeployment/lambda/alexa/power_control.py

```diff
@@ -0,0 +1,46 @@
+import os
+import boto3
+import requests
+from response import AlexaResponse
+import logging
+import config
+import utils
+logger = logging.getLogger(__name__)
+import logging
+logger = logging.getLogger(__name__)
+import time
+
+class PowerControl(object):

```

Comment:
add block comment to explain the purpose of the class

Reply:
Done